nemoguardrails/library/domain_hallucination/kb.py

```
@@ -0,0 +1,256 @@
+                with open(domain_file, "r", encoding="utf-8") as f:
+                    data = json.load(f)
+                    evidence.append(
+                        {
+                            "type": "external_kb",
+                            "source": str(domain_file),
+                            "domain": domain,
+                            "data": data,
+                        }
+                    )
+            except Exception:
+                pass
```

Ruff (0.15.15)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)